subsys/net/lib/aws_fota/Kconfig

```diff
@@ -8,6 +8,7 @@ menuconfig AWS_FOTA
 	select AWS_JOBS
 	depends on FOTA_DOWNLOAD
 	depends on CJSON_LIB
+	default y if AWS_IOT
```

Comment:
**nitpick:** this changes the behaviour of the configuration and is not related to QEMU.

Reply:
If QEMU is enabled, we want to prevent users of enabling FOTA. Otherwise, FOTA will be enabled as a default feature as long as the FOTA_DOWNLOAD library is enabled. Same for Azure.